### DISCUSSION TITLE
“Smart prefetch” for Nuxt 3?

### DISCUSSION THREAD

**Discussion by Foddy** [nuxt/nuxt#17290] [Questions] [answered]
Hi, I've always loved Nuxt 2 because of this awesome feature (among other things) described here:

https://nuxtjs.org/announcements/introducing-smart-prefetching/

In my recent tests with Nuxt 3 RC1 + Vite as a bundler, I doesn't seem to "smart prefetch" any sources from subpages which are linked with the NuxtLink component. Instead, this happens which seems to be a bug: https://github.com/nuxt/framework/issues/4475

Is this still planned, just a bug right now, a special configuration or is this feature just discontinued?

Thanks :)

**Comment by zernonia** [comment, marked as answer]
You can look at the discussion here https://github.com/nuxt/framework/issues/3649
